src/controls.lua

```diff
@@ -210,10 +210,13 @@ function Controls.loadDefaultMap()
 		f11       = {player = 1, ship = "fullsceen",    editor = nil,         menu = nil        },
 		f12       = {player = 1, ship = "debug",        editor = nil,         menu = nil        },
 		backspace = {player = 1, ship = nil,            editor = nil,         menu = "backspace"},
+
+		n         = {player = 1, ship = "debugSpawn",   editor = nil,         menu = nil        },
+		["ctrl+s"]= {player = 1, ship = "debugSave",    editor = nil,         menu = nil        },
```

Comment:
Now you can prepend "ctrl+" to a binding to make it only work when control is held down.